packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -19,10 +19,10 @@ import {
   OriginResponseEvent,
   PerfLogger
 } from "../types";
-import S3 from "aws-sdk/clients/s3";
 import { performance } from "perf_hooks";
 import { ServerResponse } from "http";
 import jsonwebtoken from "jsonwebtoken";
+import { Readable } from "stream";
```

Comment:
This is just a type, should be fine to import here.

Reply:
```suggestion
import type { Readable } from "stream";
```

Reply:
Oh, I thought Rollup would optimize this but I guess explicitly importing just type is good practice, too.